offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/GridUtils.groovy

```diff
@@ -54,6 +58,42 @@ class GridUtils {
         filterTextField.setSizeFull()
     }
 
+    /**
+     * Provides a filter field into a header row of a grid for a given column.
+     *
+     * <p>This filter tests for the provided predicate based on textual user input.</p>
+     *
+     *
+     * @param <T> the grid bean type
+     * @param <V> the column value type
+     * @param dataProvider The grid's {@link ListDataProvider}
+     * @param column The column to add the filter to
+     * @param predicate the predicate that has to be true for a tested value and a user input String
+     * @param headerRow The {@link com.vaadin.ui.components.grid.HeaderRow} of the {@link Grid}, where the filter input field is added
```

Comment:
```suggestion
     * @param <T> The grid bean type
     * @param <V> The column value type
     * @param dataProvider The grid's {@link ListDataProvider}
     * @param column The column to add the filter to
     * @param predicate The predicate that has to be true for a tested value and a user input String
     * @param headerRow The {@link com.vaadin.ui.components.grid.HeaderRow} of the {@link Grid}, where the filter input field is added
```

Reply:
?

Reply:
the parameters <T> and <V> were introduced for documentation. As far as I can see you suggest to remove them again?

Reply:
They are not part of the method.. at least I cant see a bean,...

Reply:
ok.. found it.. I oversaw that it defines T and V

Reply:
Then I just have a NIT (updated my suggestion) ;)

Reply:
per convention the javadoc starts in lower case here :)